service/src/main/java/uk/gov/hmcts/reform/fpl/service/document/transformer/RespondentStatementsTransformer.java

```diff
@@ -53,7 +52,8 @@
                 List<Element<SupportingEvidenceBundle>> furtherEvidenceDocumentsLA = new ArrayList<>();
                 List<Element<SupportingEvidenceBundle>> furtherEvidenceDocumentsNC = new ArrayList<>();
 
-                var bundleName = String.format("Respondent %d statements", count);
+                var bundleName = String.format("%s statements",
```

Comment:
Any reason for var over string?

Reply:
sonar marks it as code smell for some reason
![image](https://user-images.githubusercontent.com/3225439/122021530-a5da6b00-cdbd-11eb-87c6-1720a78f2534.png)

Reply:
We can ignore sonar in this instance if you want, that is a relatively new smell that was added and I think we have been ignoring it as it is more of a coding style than anything else. Fine to leave as is though